### PR TITLE
Bump scanamo and facia and capi clients

### LINKED ISSUE #24747: scanamo

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?
Bumps scanamo, which then in turn allows us to bump capi and facia clients.

We've done these together as the aim was to bump facia client to get captions on slideshow images. To bump `facia`, we needed to bump `capi`, which means we needed to bump `scanamo` as is has a transient dependency on `scala-collection-compat@2.1.2`, where as we need `2.6.0`.

The [`browseDependencyTree`](https://github.com/sbt/sbt-dependency-graph) plugin was a great way to find this out. Installing it globally will save lot of time in the future. 

[This fix](https://github.com/guardian/frontend/pull/24788/commits/23ce33606b5e2c347bf581577a6fa0ebf26a470d) is interesting in the sense of it wasn't failing during tests... not really sure why. 

part of: #24782 #24740

### CODE REVIEW

File: common/app/services/RedirectService.scala [hunk 1]
```diff
@@ -119,8 +117,8 @@ class RedirectService(implicit executionContext: ExecutionContext) extends GuLog
 
   def lookupRedirectDestination(source: String): Future[Option[Destination]] = {
     val fullUrl = if (source.head == '/') expectedSourceHost + source else s"$expectedSourceHost/$source"
-    ScanamoAsync
-      .exec(DynamoDB.asyncClient)(table.get('source -> fullUrl))
+    ScanamoAsync(DynamoDB.asyncClient)
+      .exec(table.get(Symbol("source") -> fullUrl))
```
Comment (jamesgorrie) [inline common/app/services/RedirectService.scala:121]: This doesn't work

File: project/Dependencies.scala [hunk 2]
```diff
@@ -40,7 +40,7 @@ object Dependencies {
   val jodaTime = "joda-time" % "joda-time" % "2.9.9"
   val jodaConvert = "org.joda" % "joda-convert" % "1.8.3"
   val jSoup = "org.jsoup" % "jsoup" % "1.10.3"
-  val json4s = "org.json4s" %% "json4s-native" % "3.5.3"
+  val json4s = "org.json4s" %% "json4s-native" % "4.0.4"
```
Comment (OllysCoding) [inline project/Dependencies.scala:43]: Update to json4s fixed issues with `Future` timeouts in tests:
image
image

File: project/Dependencies.scala [hunk 3]
```diff
@@ -66,7 +66,7 @@ object Dependencies {
 
   val kinesisLogbackAppender = "com.gu" % "kinesis-logback-appender" % "1.4.0"
   val targetingClient = "com.gu" %% "targeting-client-play26" % "0.14.7"
-  val scanamo = "com.gu" %% "scanamo" % "1.0.0-M8"
+  val scanamo = "org.scanamo" %% "scanamo" % "1.0.0-M11"
```
Comment (DavidLawes) [inline project/Dependencies.scala:69]: Did the latest ([1.0.0-M19](https://mvnrepository.com/artifact/org.scanamo/scanamo_2.13/1.0.0-M19)) cause problems? Either way, 1.0.0-M11 is compatible with scala 2.13 so I think this current PR would close https://github.com/guardian/frontend/issues/24747

**Review by OllysCoding — APPROVED** [review]

File: project/Dependencies.scala [hunk 4]
```diff
@@ -5,8 +5,8 @@ import sbt._
 object Dependencies {
   val identityLibVersion = "3.240-C5"
   val awsVersion = "1.11.240"
-  val capiVersion = "17.22.1"
-  val faciaVersion = "3.3.8"
+  val capiVersion = "17.24.1"
```
Comment (mxdvl) [inline project/Dependencies.scala:8]: Probably the root cause of https://github.com/guardian/dotcom-rendering/issues/4343

**Pull request merged**